include/seqan3/range/views/minimiser.hpp

```diff
@@ -136,18 +136,9 @@ class minimiser_view : public std::ranges::view_interface<minimiser_view<urng_t>
      *
      * No-throw guarantee.
      */
-    auto end()
-    {
-        return std::ranges::end(urange);
-    }
-
-    //!\copydoc end()
     auto end() const
-    //!\cond
-        requires seqan3::const_iterable_range<urng_t>
```

Comment:
The `end const` version now has no requires clause, but the `cend()` function does?

Reply:
I removed it